dotnet/src/Functions/Functions.OpenApi/OpenApiKernelPluginFactory.cs

```diff
@@ -0,0 +1,344 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.Http;
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi;
+
+/// <summary>
+/// Provides static factory methods for creating OpenAPI KernelPlugin implementations.
+/// </summary>
+public static partial class OpenApiKernelPluginFactory
+{
+    /// <summary>
+    /// Creates a plugin from an OpenAPI v3 endpoint.
```

Comment:
I think the summary of this method and the other ones should be rephrased not say anything about spec version and endpoint.